docs/myst.yml

```diff
@@ -24,6 +24,7 @@ project:
     - file: background.md
     - file: data.md
     - file: methodology.md
+    - file: long_term_projections.md
     - file: local_area_calibration_setup.ipynb
     - file: long_term_projections.ipynb
```

Comment:
I need to learn more about the markdown versions of these notebooks. Assuming you're switching from ipynb to md, do you need to remove `long_term_projections.ipynb` from the myst.yml also?